stencil-library/CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file. Dates are d
 
 Generated by [`auto-changelog`](https://github.com/CookPete/auto-changelog).
 
+#### 3.1.0
+> 1 May 2023
+
+- Add `single-line` story for `CardForm`
```

Comment:
I don't think we should bump the version just for stories since it will publish a whole new npm package even though the only thing that really changed was documentation

Reply:
Sounds good to me - thanks for the heads up. I'll need to get some more practice determining what is worthy of a version update or not.